website/server/controllers/api-v3/auth.js

```diff
@@ -231,7 +231,7 @@ api.loginLocal = {
 
 function _passportProfile (network, accessToken) {
   if (network in SUPPORTED_SOCIAL_NETWORKS) {
-    return new Promise.promisify((resolve, reject) => {
+    return new Bluebird((resolve, reject) => {
```

Comment:
I think that happens because later you're `await`ing passportProfile but it when `network in SUPPORTED_SOCIAL_NETWORKS ` returns false it doesn't return a promise that is necessary when using await .

Also, I don't know if we misunderstood but I meant that you should use `new Promise(...)` not `new Promise.promisify`